### ISSUE TITLE
Multiple input fields - accessibility 

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#184]
From e-QIP-prototype-truetandem created by [Jay-Neighbours-I72](https://github.com/Jay-Neighbours-I72) : ryanhofdotgov/e-QIP-prototype-truetandem#2738  

Inputs with multiple fields (SSN, phone numbers) - need to be clear to screen readers which input they are in - example - "first 3 numbers, middle 3 numbers, last 4 numbers"
![screen shot 2018-03-19 at 3 05 48 pm](https://user-images.githubusercontent.com/19935974/37616652-05f463a6-2b87-11e8-8499-d8d393c98f31.jpg)

**Comment by afeld** [comment]
It looks like there's an `aria-label` attribute, which [should be sufficient](https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-label). Is the issue that the `<label>` is empty and therefore overriding it?

<img width="531" alt="screen shot 2018-06-12 at 3 40 04 pm" src="https://user-images.githubusercontent.com/86842/41313118-07d5448e-6e57-11e8-89a8-b32b7d4bfa4f.png">
